coderd/database/queries/presets.sql

```diff
@@ -16,8 +18,9 @@ VALUES (
 
 -- name: InsertPresetParameters :many
 INSERT INTO
-	template_version_preset_parameters (template_version_preset_id, name, value)
+	template_version_preset_parameters (id, template_version_preset_id, name, value)
```

Comment:
Why is it important to be able to control the ID of a template version preset?

Reply:
when setting up test environments where one needs to assert based on the relationships between presets, their parameters and other parts of the DB, we need to be able to express that relationship in the test by setting the ids explicitly.